src/bridge.c

```diff
@@ -149,7 +149,8 @@ zmqBridge_close(mamaBridge bridgeImpl)
       /* The timer thread expects us to be responsible for terminating it */
       wthread_join(timerThread, NULL);
    }
-    mamaBridgeImpl_setClosure(bridgeImpl, NULL);
+   mamaBridgeImpl_setClosure(bridgeImpl, NULL);
+   free(closure);
```

Comment:
Is this part of the error message changes or something separate?

Reply:
This was a fix made in early June that has not been merged into master, which is why it's showing up here.